ols/src/llms/utils.py

```diff
@@ -0,0 +1,42 @@
+from langchain.callbacks.base import BaseCallbackHandler
```

Comment:
I'd rather not to use `utils.py  again please. It contains implementation of `GenericTokenCounter`

Reply:
I would propose something like
```
ols
  /metrics
     metrics.py   # moved from ols/app
     token_counter.py
```

Reply:
will move to `ols/metrics/token_counter.py`